Maestro/subscriptions.json

```diff
@@ -135,6 +141,26 @@
             "/p:NotifyGitHubUsers=dotnet/coreclr-contrib",
             "/verbosity:Normal"
           ]
+        },
+        // This handler will bring the Latest CoreFX master build into WCF master
+        {
+          "maestroAction": "wcf-general",
+          "maestroDelay": "00:10:00",
+          "ScriptFileName": "run.cmd",
+          "Arguments": [
+            "build",
+            "-Project='tests\\build.proj'",
```

Comment:
I forgot, another difference is that there is no `run build` in the wcf config.json--only the coreclr one has that. It's `build-managed.cmd` in wcf and corefx. (Aka `run build-managed`, although to be consistent the batch file would be better here.)